bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -252,19 +250,20 @@ private func dummy() {
     _ = NSLocalizedString("Select a color", comment: "")
     _ = NSLocalizedString("Select a color above", comment: "")
     _ = NSLocalizedString("Select a layout", comment: "")
-    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select font size option value e.g: \"Selected: Large\".\ntranslators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: Default", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.native.js#L111:L111)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.